src/functions/styling.js

```diff
@@ -30,11 +30,15 @@ defineFunction({
         parser.consumeSpaces();
         const body = parser.parseExpression(true, breakOnTokenText);
 
+        // TODO: Refactor to avoid duplicating styleMap in multiple places (e.g.
+        // here and in buildHTML and de-dupe the enumeration of all the styles).
+        // $FlowFixMe: The names above exactly match the styles.
+        const style: StyleStr = funcName.slice(1, funcName.length - 5);
```

Comment:
Can you explain in more detail what the flow error was here?

Reply:
Gotcha.  It would be nice to limit the slicing to remove leading `\` as much as possible.  Not sure how much that would help with this flow error though.